### PR TITLE
added a small clarification to the docs

### PR DESCRIPTION
**Pull request by daniel-sintef**
# Description:
Small clarification to the docs regarding the manner in which strategies are implemented. 

## Type of change:

- [ ] Bug fix.
- [ ] New feature.
- [X] Documentation update.

## Checklist for the reviewer:

This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ OTEAPI Core provides the core functionality of OTEAPI, which stands for the *Ope
 
 It uses the [strategy](https://en.wikipedia.org/wiki/Strategy_pattern) software design pattern to implement a simple and easy to extend access to a large range of data resources.
 Semantic interoperability is supported via mapping of data models describing the data to ontologies.
+We provide a set of abstract interfaces for strategies and data models used in their configuration.
+This repo also contains implementations for several standard strategies, e.g., downloading files, parsing Excel documents.
```
Comment (francescalb) [inline README.md:21]:
```suggestion
A set of strategy interfaces that can be considered abstract classes for the implementation of strategies,  and data models used in their configuration, are provided.
This repo also contains implementations for several standard strategies, e.g., downloading files, parsing Excel documents.
```

File: docs/index.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ OTEAPI Core provides the core functionality of OTEAPI, which stands for the *Ope
 
 It uses the [strategy](https://en.wikipedia.org/wiki/Strategy_pattern) software design pattern to implement a simple and easy to extend access to a large range of data resources.
 Semantic interoperability is supported via mapping of data models describing the data to ontologies.
+We provide a set of abstract interfaces for strategies and data models used in their configuration.
```
Comment (francescalb) [inline docs/index.md:20]: I do not think that we are supposed to make the changes in the docs part?

Comment (daniel-sintef) [reply docs/index.md]: I'm not sure, the tests fail unless I update it in both places...

File: README.md [hunk 3]
```diff
@@ -17,6 +17,8 @@ OTEAPI Core provides the core functionality of OTEAPI, which stands for the *Ope
 
 It uses the [strategy](https://en.wikipedia.org/wiki/Strategy_pattern) software design pattern to implement a simple and easy to extend access to a large range of data resources.
 Semantic interoperability is supported via mapping of data models describing the data to ontologies.
+A set of strategy interfaces that can be considered abstract classes for the implementation of strategies,  and data models used in their configuration, are provided.
```
Comment (jesper-friis) [inline README.md:20]: I am not sure this is so elightening. oteapi-core is actually not using abstract base classes, but rather protocol classes.

File: docs/index.md (hunk 2 above)
Comment (jesper-friis) [reply docs/index.md]: Yes, I think you have to update both places, but pre-commit will update the docs for you.

If you aren't using pre-commit, install it with

    pip install pre-commit
    pre-commit install

File: README.md (hunk 3 above)
Comment (CasperWA) [inline README.md:20]:
```suggestion
A set of strategy interfaces that can be considered abstract classes for the implementation of strategies, and data models used in their configuration, are provided.
```

File: docs/index.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ OTEAPI Core provides the core functionality of OTEAPI, which stands for the *Ope
 
 It uses the [strategy](https://en.wikipedia.org/wiki/Strategy_pattern) software design pattern to implement a simple and easy to extend access to a large range of data resources.
 Semantic interoperability is supported via mapping of data models describing the data to ontologies.
+A set of strategy interfaces that can be considered abstract classes for the implementation of strategies,  and data models used in their configuration, are provided.
```
Comment (CasperWA) [inline docs/index.md:20]:
```suggestion
A set of strategy interfaces that can be considered abstract classes for the implementation of strategies, and data models used in their configuration, are provided.
```

Comment (CasperWA) [reply docs/index.md:20]: This change needs to be committed as well for the `pre-commit` CI to succeed.

**Review by CasperWA — APPROVED** [review]

**Pull request merged**